notebooks/gridded_stitching_regular-py.py

```diff
@@ -0,0 +1,208 @@
+
+# load libraries
+import xarray as xr
+import fsspec as fsspec
+import pandas as pd
+import numpy as np
+from matplotlib import pyplot as plt
+import seaborn as sns
+import intake
+
+
```

Comment:
So the first three functions defined here are all about working with pangeo data, they would probably be useful in some script that defined pangeo helper functions and or netcdf helper functions. 

actually, do we already have these functions defined somewhere 🤦‍♀️

Reply:
I'm not sure if we have them defined already, but we should. I imagine that will be among the first changes we make once we start translating the R code into python. I'm capturing it in an issue now